eclair-core/src/test/scala/fr/acinq/eclair/channel/states/h/ClosingStateSpec.scala

```diff
@@ -775,6 +791,49 @@ class ClosingStateSpec extends TestkitBaseClass with StateTestsHelperMethods {
     awaitCond(alice.stateName == CLOSED)
   }
 
+
+  test("recv BITCOIN_TX_CONFIRMED (future remote commit, option_static_remotekey)", Tag("static_remotekey")) { f =>
```

Comment:
I think this one should be updated with the additional timeout HTLCs, like the "normal" `future remote commit` test.
Ideally if you could extract most of it to a shared function that both tests would call, and only do the last part (sending the `WatchEventSpent`s) that's custom in each test it would be nice.

Reply:
Done in 4abf88d